src/ring_theory/finiteness.lean

```diff
@@ -58,9 +58,16 @@ instance is_noetherian.finite [is_noetherian R M] : finite R M :=
 ⟨is_noetherian.noetherian ⊤⟩
 
 namespace finite
+open submodule set
 
 variables {R M N}
 
+lemma exists_fin [finite R M] : ∃ (n : ℕ) (s : fin n → M), span R (range s) = ⊤ :=
+begin
+  rw ← submodule.fg_iff_exists_fin_generating_fam,
+  exact module.finite.out
+end
```

Comment:
Probably no harm in golfing this one, since there's nothing being proved here and it's just unpacking a typeclass into another lemma
```suggestion
submodule.fg_iff_exists_fin_generating_fam.mp out
```